tests/end2end-tests/graphstorm-nc/test.sh

```diff
@@ -53,6 +53,12 @@ python3 -m graphstorm.run.gs_node_classification --workspace $GS_HOME/training_s
 
 error_and_exit $?
 
+echo "**************dataset: MovieLens, RGCN layer: 1, node feat: fixed HF BERT, BERT nodes: movie, inference: full-graph, ngnn layer between GNN layer: 1"
+python3 -m graphstorm.run.gs_node_classification --workspace $GS_HOME/training_scripts/gsgnn_np --num-trainers $NUM_TRAINERS --num-servers 1 --num-samplers 0 --part-config /data/movielen_100k_train_val_1p_4t/movie-lens-100k.json --ip-config ip_list.txt --ssh-port 2222 --cf ml_nc.yaml --use-mini-batch-infer false --num-ngnn-layer 1 --save-model-path ./models/movielen_100k/train_val/movielen_100k_ngnn_model
```

Comment:
You need to add `error_and_exit $?` after running `python3 -m graphstorm.run.gs_node_classification`. I will check whether the training fails.

Reply:
Got it. I would add one more error_and_exit $? here.